execute_pipeline.py

```diff
@@ -145,6 +145,8 @@ def _load_config_spec(config_spec, config_sections, repl_vars):
         all_config_data = yaml.load(config_file)
     for section in config_sections:
         data.update(all_config_data[section])
+    if language in all_config_data:
```

Comment:
In the case that a list of config sections is specified on the commend line (after a ':' symbol) this will still try to load the language section (and will do so twice if it is listed).